pallets/funding/src/lib.rs

```diff
@@ -18,96 +18,53 @@
 //!
 //! Polimec's main business logic. It allows users to create, evaluate, and fund projects.
 //!
-//! It rewards project evaluators and contributors with `Contribution Tokens`. These tokens
-//! can be redeemed for a project's native tokens, after their parachain is deployed on mainnet.
-//! ## 👷 Work in progress 🏗️
-//! Expect major changes between PRs
+//! Participants get contribution tokens. Contribution tokens are tokens issued by projects which successfully raised
+//! funds on Polimec. They are distributed to evaluators and participants who contributed to the project’s successful
+//! funding round. Contribution tokens are transferability-locked in the wallet of participants or evaluators after
+//! distribution and are automatically converted to the project’s transferable mainnet token at launch.
 //!
 //! ## Overview
-//! The official logic for Polimec's blockchain can be found at our [whitepaper](https://polimec.link/whitepaper).
+//! The official logic for Polimec's blockchain can be found in our [knowledge hub](https://hub.polimec.org/).
 //!
 //! There are 3 types of users in Polimec:
 //! - **Issuers**: They create projects and are responsible for their success.
-//! - **Evaluators**: They evaluate projects and are rewarded for their work.
-//! - **Contributors**: They contribute financially to projects and are rewarded on the basis of their contribution
+//! - **Evaluators**: They are incentivized to assess projects accurately by locking their PLMC. If enough PLMC is locked,
+//! a project is given access to the funding round. Evaluators are either rewarded in contribution tokens if the project gets funded,
+//! or have their PLMC slashed otherwise.
+//! - **Participants**: They contribute financially to projects by locking PLMC and paying out USDT/USDC/DOT, and are rewarded in Contribution Tokens.
```

Comment:
for consistency reasons, would write "contribution tokens" (small letters, not capital ones)